test/recipes/80-test_cms.t

```diff
@@ -641,7 +641,7 @@ subtest "CMS Decrypt message encrypted with OpenSSL 1.1.1\n" => sub {
 
     SKIP: {
         skip "EC isn't supported in this build", 1
```

Comment:
Should you also changed this description to something like "EC or DES isn't supported..."

Reply:
Good point. Fixed.